app/policies/list_policy.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ListPolicy < ApplicationPolicy
```

Comment:
Is this ever used by Doorkeeper? I couldn't find any indication that doorkeeper checked pundit policies....